pystac_client/client.py

```diff
@@ -33,13 +33,14 @@ def open(
         headers: Dict[str, str] = None,
         parameters: Optional[Dict[str, Any]] = None,
         ignore_conformance: bool = False,
-    ) -> "Client":
+        media_type: Optional[Union[str, pystac.MediaType]] = None) -> "Client":
```

Comment:
```suggestion
    ) -> "Client":
```
If we use `pystac.MediaType.GEOJSON` in the call to `Catalog.get_links` below then this parameter should no longer be needed.